packages/hub/routes/scheduled-payments.ts

```diff
@@ -139,6 +135,11 @@ export default class ScheduledPaymentsRoute {
         errors: serializeErrors(errors),
       };
     } else {
+      if (params.recurringDayOfMonth != null && params.recurringUntil != null) {
+        params.payAt = calculateNextPayAt(new Date(), params.recurringDayOfMonth, params.recurringUntil);
+        if (!params.payAt) throw new Error(`cannot calculate next payment date`);
```

Comment:
1. The error message is too vague (would be good to have a reason why it cannot be calculated)
2. Won't the request will fail with a 500 Internal Server error if you throw an error here? In that case it is a better idea to put this check in the validator so that the client knows what went wrong.